infra/src/main/resources/i18n/translations.fi.json

```diff
@@ -192,103 +192,121 @@
             "UNVERIFIED_DESIGNED_GEOMETRY": "Epäluotettava suunnitelma",
             "UNKNOWN": "Ei tiedossa"
         },
-        "elevation-measurement-method": {
+        "ElevationMeasurementMethod": {
             "TOP_OF_SLEEPER": "Korkeusviiva",
             "TOP_OF_RAIL": "Kiskon selkä",
             "UNKNOWN": "Ei tiedossa",
-            "UNKNOWN-WITH-TERM-CONTEXT": "Korkeusasema ei tiedossa"
+            "UNKNOWN_WITH_TERM_CONTEXT": "Korkeusasema ei tiedossa"
         },
-        "switch-hand": {
+        "SwitchHand": {
             "LEFT": "Vasen",
             "RIGHT": "Oikea",
             "NONE": "Ei kätisyyttä"
         },
-        "location-track-type": {
+        "LocationTrackType": {
             "MAIN": "Pääraide",
             "SIDE": "Sivuraide",
             "TRAP": "Turvaraide",
             "CHORD": "Kujaraide",
             "UNKNOWN": "Ei tiedossa"
         },
-        "publish-operation": {
+        "Operation": {
             "CREATE": "Luonti",
             "DELETE": "Poisto",
             "MODIFY": "Muokkaus",
             "RESTORE": "Palautus",
             "CALCULATED": "Laskennallinen muutos"
         },
-        "topological-connectivity-type": {
+        "TopologicalConnectivityType": {
             "NONE": "Ei kytketty",
             "START": "Raiteen alku",
             "END": "Raiteen loppu",
             "START_AND_END": "Raiteen alku ja loppu"
         },
-        "alignment-end-type": {
-            "SWITCH": "Vaihde",
-            "LOCATION_TRACK": "Jatkuu toisena raiteena",
-            "ENDPOINT": "Raide päättyy",
-            "UNKNOWN": "Ei asetettu"
-        },
-        "location-track-state": {
+        "LocationTrackState": {
             "BUILT": "Rakennettu",
             "IN_USE": "Käytössä",
             "NOT_IN_USE": "Käytöstä poistettu",
             "DELETED": "Poistettu"
         },
-        "layout-state": {
+        "LayoutState": {
             "IN_USE": "Käytössä",
             "NOT_IN_USE": "Käytöstä poistettu",
             "DELETED": "Poistettu"
         },
-        "layout-state-category": {
+        "LayoutStateCategory": {
             "EXISTING": "Olemassa oleva kohde",
             "NOT_EXISTING": "Poistunut kohde"
         },
-        "ratko-push-error-type": {
+        "RatkoPushErrorType": {
             "PROPERTIES": "ominaisuustietojen",
             "LOCATION": "sijainnin",
             "GEOMETRY": "geometrian",
             "STATE": "tilan"
         },
-        "ratko-push-error-operation": {
+        "RatkoPushErrorOperation": {
             "CREATE": "luonti",
             "UPDATE": "päivitys",
             "DELETE": "poisto",
             "FETCH_EXISTING": "pohjatietojen hakeminen Ratkosta"
         },
-        "trap-point": {
-            "Yes": "Kyllä",
-            "No": "Ei",
-            "Unknown": "Ei tiedossa"
+        "TrapPoint": {
+            "YES": "Kyllä",
+            "NO": "Ei",
+            "UNKNOWN": "Ei tiedossa"
```

Comment:
Nämä ovat mielestäni olleet tarkoituksella näin (muistelen itsekin ihmetelleeni tätä joskus). Olisikohan syynä ollut joku ulkoinen formaatti, tyyliin että nämä tulivat suoraan inframallien XML:istä noin... voi olla että joku muu muistaisi paremmin 🤔 Anyhoo, kannattanee testata tämän muutoksen jälkeen että monimutkaisemmat inframallien tuomiset Geoviitteeseen, Ratko-viennit jne. toimivat oikein

Reply:
🤔 Tässä on muutettu enumin varianttien nimiä, joiden pitäisi vaikuttaa vain ts-koodiin. Enumin arvot ovat olleet numeroita (esim. TrapPoint.Yes = 1 ajon aikana)

Kuulostaisi myös hyvin merkilliseltä jos käännösavaimia on käytetty muuallakin johonkin varsinaisiin mäppäyksiin eikä pelkästään vaan ui:lla näkyviin käännöksiin

Reply:
Ei vaan jaap juup, bäkkärillä tuo toki saattaakin aiheuttaa problematiikkaa, pitääpä chekkailla